.changes/unreleased/Fixed-20231107-140052.yaml

```diff
@@ -0,0 +1,5 @@
+kind: Fixed
```

Comment:
I don't think a change fragment is required, there is a special action I can run to update the website without a release.

Reply:
Removed.